crates/reporting/src/cli.rs

```diff
@@ -33,7 +33,7 @@ impl Problems {
         const YELLOW: &str = ANSI_STYLE_CODES.yellow;
         const RESET: &str = ANSI_STYLE_CODES.reset;
 
-        println!(
+        print!(
```

Comment:
We used to be printing 

```
3 errors and 0 warnings found in <ignored for test> ms
.
```

now we get

```
3 errors and 0 warnings found in <ignored for test> ms.
```

Reply:
I tried keeping `println!` and putting the period at the end of this statement, but there are multiple places where we append text after this line that would break if we always put the period here. For another PR 🤷🏼